Makefile

```diff
@@ -75,20 +75,22 @@ reqs_dev: requirements/requirements_dev.txt
 $(env_tag): requirements/requirements.txt
 	@echo "==Installing requirements.txt=="
 	pip-sync --quiet requirements/requirements.txt
-	rm -f $(env_dev_tag)
-	rm -f $(install_tag)
+	rm $(install_tag)
 	touch $(env_tag)
 
 $(env_dev_tag): requirements/requirements_dev.txt
 	@echo "==Installing requirements_dev.txt=="
 	pip-sync --quiet requirements/requirements_dev.txt
-	rm -f $(env_tag)
-	rm -f $(install_tag)
+	rm $(install_tag)
 	touch $(env_dev_tag)
 
 setup: $(env_tag)
+	@echo "Setup"
+	rm -f $(env_dev_tag)
 
 setup_dev: $(env_dev_tag)
+	@echo "Setup Dev"
+	rm -f $(env_tag)
```

Comment:
Wouldn't be nicer if we keep the same style in messages printed to screen?

In these cases I'd use: `"==Setting up package environment=="` and `"==Setting up development environment=="`

Reply:
+1